tests/Feature/ServiceRefreshTest.php

```
@@ -0,0 +1,430 @@
+    private function renderedScript(): string
+    {
+        return view('ssh.os.refresh-services', $this->viewData())->render();
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    private function viewData(): array
+    {
+        $probe = new ProbeServices;
+        $reflection = new \ReflectionClass($probe);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '420', column '27'). (undefined)

(MissingImport)